### ISSUE TITLE
Add link to source code of each model to model documentation (Trac #883)

### ISSUE THREAD

**Issue by ajj** [SasView/sasmodels#146]
The documentation for each model distributed in sasmodels should have a link to the source code on github, ideally to the revision that ships.

This will make it easier for users to find the source code to base their own models on without having to dig through the distributed packages.

Migrated from http://trac.sasview.org/ticket/883
```json
{
    "status": "closed",
    "changetime": "2019-03-25T18:04:28",
    "_ts": "2019-03-25 18:04:28.973651+00:00",
    "description": "The documentation for each model distributed in sasmodels should have a link to the source code on github, ideally to the revision that ships.\n\nThis will make it easier for users to find the source code to base their own models on without having to dig through the distributed packages.",
    "reporter": "ajj",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2017-03-15T14:23:35",
    "component": "sasmodels",
    "summary": "Add link to source code of each model to model documentation",
    "priority": "minor",
    "keywords": "",
    "milestone": "SasView 4.3.0",
    "owner": "smk78",
    "type": "enhancement"
}
```

**Comment by RichardHeenan** [comment]
Trac update at `2017/03/16 11:06:44`: **richardh** commented:

If the docs had an appropriate instruction and/or a pointer, and also a big note in the plugins directory, is it that hard to find the directory where the source code from the download lives? (On my windows, program files(x86)/sasview41/sasmodels-data/models) This works without web access to github and is the correct revision for my download. 

OK the user might discover, I think, that they can add or modify models there without using their plugins directory, so we might want to ban that?

**Comment by pkienzle** [comment]
Trac update at `2017/03/16 14:15:42`: **pkienzle** commented:

Or provide a "clone existing model" button in the plugin editor window?

**Comment by butlerpd** [comment]
Trac update at `2017/10/27 17:15:26`: **butler** changed milestone from "SasView 4.2.0" to "SasView 4.3.0"

**Comment by smk78** [comment]
Trac update at `2019/03/25 10:53:12`: **smk78** commented:

Replying to http://trac.sasview.org/ticket/883#comment:1 :
The start of plugin.rst actually says:

```text
The built-in modules are available in the models subdirectory of the sasmodels package. 
For SasView on Windows, these will be found in C:\Program Files (x86)\SasView\sasmodels-
data\models. On Mac OSX, these will be within the application bundle as /Applications/
SasView 4.0.app/Contents/Resources/sasmodels-data/models.
```

**Comment by smk78** [comment]
Trac update at `2019/03/25 18:04:28`: 
* **smk78** commented:

In changeset 0507e09b26f5789862db8a175d64657cf4f3b829:

```text
#!CommitTicketReference repository="sasmodels" revision="0507e09b26f5789862db8a175d64657cf4f3b829"
Added link to source code to each model. Closes #146
```


* **smk78** changed owner from "" to "smk78"
* **smk78** changed resolution from "" to "fixed"
* **smk78** changed status from "new" to "closed"
